qiskit/transpiler/_transpiler.py

```diff
@@ -207,70 +193,61 @@ def compile_circuit(quantum_circuit, basis_gates='u1,u2,u3,cx,id', coupling_map=
             second element being the layout.
 
     Raises:
-        QISKitCompilerError: if the format is not valid.
+        TranspilerError: if the format is not valid.
```

Comment:
Again, double check the function is actually raising only this type of exception or change the docstring accordingly.

Reply:
fixed in https://github.com/QISKit/qiskit-core/pull/578/commits/d50505dbb3cde63555637d5337ad019fb43ececc